packages/node-cli/src/core/CommandActionExecutor.js

```diff
@@ -138,7 +136,7 @@ module.exports = class CommandActionExecutor {
 			const overriddenCommandArguments = beforeExecutingOutput.arguments;
 
 			const argumentsFromQuestions =
-				runInInteractiveMode || command._commandMetadata.forceInteractiveMode
+				runInInteractiveMode /*|| command._commandMetadata.forceInteractiveMode*/
```

Comment:
Is there a reason why the commented piece of code can't be removed yet?

Reply:
I forgot to delete it. The code can't be removed, so I'll restore it